### ISSUE TITLE
Regression: Table movement broken on dev

### ISSUE THREAD

**Issue by hq6** [vimwiki/vimwiki#656]
This issue is reproducible on the `dev` branch in the default syntax with the minimum configuration described in the Troubleshooting page.

### Steps to Reproduce.

1. Copy the following table into a wiki file.
```
| column 1 | column 2        | column3 |
|----------|-----------------|---------|
|          | fefeafe fefiefe |         |
|          | efef            |         |
|          |                 |         |
```
2. Move the cursor to the string `column 1` in normal mode.
3. Hit Tab three times. Observe that the cursor is now in the middle of `column3` at the character `u` instead of at the character `c`.  **This is the first problem.**
4. Hit <Shift-Tab> as many times as you like. Observe that the cursor fails to move backwards. **This is the second problem**

@lyokha, I think this was introduced in 5e4a89c89889c2e630cc97c620bc138c7efa5adf or one of the other commits in #47. If you could take a look, that would be greatly appreciated.

**Comment by lyokha** [comment]
I removed inline formatting (shrinking) of the current line in `get_rows()` that was introduced to avoid unnecessary traversals of the whole table when checking if it must be re-formatted. Now this shrinking occurs in `get_aligned_rows()` where it's directly needed with a temporary line.

**Comment by hq6** [comment]
Fixed in ae3db7a079d6c9cb8b43a7276c1695fabdab9509.
